guide/README.md

```diff
@@ -0,0 +1,285 @@
+
+## Force matching (FM)
+
+All files fore running the actual force matching calculation can be found in  `csg-tutorials/spce/force_matching/`. Have a look at the settings file (`fmatch.xml`). In the general force matching section (`cg.fmatch`), the number of frames to read in simultaneously (`frames_per_block`) and the type of LS solver (constrainedLS) are fixed (it is preferred to yous constrained LS). The fmatch block of the interaction (`cg.non-bonded.fmatch`) contains the interaction range (`min` and `max`), the step size for the internal spline representation (`step`) and the output step (`out_step`). `min` and `max` and have to be adjusted to be within the range of the RDF (see the calculating of RDF section of the IBI tutorial). Run the FM calculation (see also the corresponding line of the `run.sh` script): 
+
+```bash
+csg_fmatch --top ../atomistic/topol.tpr --trj ../atomistic/traj.trr --options fmatch.xml --cg water.xml (--begin $equi ) # ( ) denotes additional option: , --begin # time frame in GROMACS units (ps) to start analysis (e.g. 1 to 5)
+```
+
+To obtain the CG potential, the CG force has to be integrated. (see the appropriate lines in the `run.sh` script):
+
+```bash
+csg_call table integrate CG-CG.force CG-CG.pot # integrates the table
+csg_call table linearop CG-CG.pot CG-CG.pot -1 0 # multiplication of all table values by -1 (potential)
+```
+
+Change the spline grid (`step`), blocksize and parameter constrainedLS. This should get a feeling for the whole procedure.
+
+## Running of CG simulation
+
+To run a CG simulation with GROMACS, the potential has to be converted to a potential table, GROMACS can read (`table_CG_CG.xvg`). (Check the inverse section in the `fmatch.xml` for the corresponding options):
+
+```bash
+csg_call --options fmatch.xml --ia-name CG-CG --ia-type non-bonded convert_potential gromacs --clean input.pot table_CG_CG.xvg # calls convert_potential gromacs. Unsampled regions for distances smaller than the min value are extrapolated.
+```
+
+To run a CG simulation, you will need the `conf.gro`, `topol.top`, `index.ndx` and `grompp.mdp` files. You can use the ones of the ibi tutorial and adjust the number of timesteps and output settings. Then run the simulation. Afterwards, you can calculate the RDF and thermodynamic data as explained in the IBI tutorial. You can also use the `Extract_Energies.sh` script of the ibi tutorial. When calculating the RDF keep in mind that now you don't need a mapping file and the --cg option can be omitted.
```

Comment:
"You can use the ones of the ibi tutorial and ..." -> "You can use the ones from the IBI tutorial, and ..."

"... script of the ibi tutorial." -> "... script of the IBI tutorial."

"When calculating the RDF keep in mind that now you don't ..." -> "When calculating the RDF from the CG simulation, you don't ..."